src/net_processing.cpp

```diff
@@ -4929,6 +5133,16 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
                     if (ping_time.count() >= 0) {
                         // Let connman know about this successful ping-pong
                         pfrom.PongReceived(ping_time);
+                        if (pfrom.IsPrivateBroadcastConn()) {
+                            m_tx_for_private_broadcast.BroadcastEnd(pfrom.GetId(), /*confirmed_by_node=*/true);
```

Comment:
07dee91112189be02c6694385de261570725b56b `BroadcastEnd()` returns a `bool`, seems like you could include that in the log message where you have the word "probably"

Reply:
By "probably successful private broadcast" I meant that probably the transaction will successfully reach the network, not that we successfully gave it to one peer. Reworded.

Also the return value of `BroadcastEnd()` was not used by the callers. Changed it to `void`.